templates/launch.html

```diff
@@ -1,215 +1,405 @@
+                                type="button"
+                                class="ons-btn"
+                                onclick="loadMetadataForRemoteSchema()">
+                            <span class="ons-btn__inner"><span class="ons-btn__text">Load Metadata</span></span>
+                        </button>
+                    </div>
+                    <hr>
+                    <h2 class="ons-u-mt-m">Survey Actions</h2>
+                    <div>
+                        <button id="submit-btn"
+                                type="submit"
+                                class="ons-btn ons-btn--disabled ons-u-ml-xs ons-u-mb-s"
+                                disabled
+                                onclick="saveResponseId()">
+                            <span class="ons-btn__inner"><span class="ons-btn__text">Open Survey</span></span>
+                        </button>
+                        <button name="action_flush"
+                                id="flush-btn"
+                                type="submit"
+                                value="Flush Survey Data"
+                                disabled
+                                class="ons-btn ons-btn--disabled ons-u-mb-s">
+                            <span class="ons-btn__inner"><span class="ons-btn__text">Flush Survey Data</span></span>
+                        </button>
+                        <button id="local-storage-btn"
+                                type="button"
+                                class="ons-btn ons-u-mb-s"
+                                onclick="clearLocalStorage()">
+                            <span class="ons-btn__inner"><span class="ons-btn__text">Clear Local Storage</span></span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+            <div class="ons-grid__col ons-col-6@xl">
+                <div class="ons-pl-grid-col">
+                    <div id="metadata-accordion" class="ons-accordion">
+                        <button type="button"
+                                class="ons-btn ons-accordion__toggle-all ons-u-mb-s ons-u-d-no ons-btn--small"
+                                data-close-all="Hide all"
+                                data-group="metadata-accordion">
+                            <span class="ons-btn__inner ons-accordion__toggle-all-inner"><span class="ons-btn__text">Show all</span></span>
+                        </button>
+                    </div>
+                    <div id="survey-type-metadata-accordion"
+                         class="ons-details ons-js-details ons-details--accordion ons-u-vh"
+                         data-group="metadata-accordion">
+                        <div class="ons-details__heading ons-js-details-heading"
+                             role="button"
+                             id="survey_type_metadata_detail">
+                            <h2 class="ons-details__title survey_heading"></h2>
```

Comment:
The reason it's initially empty is so that the business/health survey metadata titles can be dynamic when they need to be shown. The JS will insert the text when you load metadata via CIR (given the type you choose is business or health).

Reply:
Ah makes sense, thanks 👍

Reply:
I added some text within the h2 tags so it's easier to read, and this will be overwritten when the JS is applied 👍